src/converter/entities.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  createIntegrationEntity,
+  getTime,
+  convertProperties,
+  Entity,
+} from '@jupiterone/integration-sdk';
+import { getSeverity } from './utils';
+
+export const getAccountEntity = (instance: any): Entity => ({
+  _key: `bugcrowd:account:${instance.id}`,
+  _type: 'bugcrowd_account',
+  _class: ['Account'],
+  name: instance.name,
+  displayName: instance.name,
+  description: instance.description,
+});
+
+export const getServiceEntity = (instance: any): Entity => ({
+  _key: `bugcrowd:service:${instance.id}:bug-bounty`,
+  _type: 'bugcrowd_service',
+  _class: ['Service', 'Control'],
+  name: 'BugCrowd Security Testing',
+  displayName: 'BugCrowd Security Testing',
+  description: 'Crowd-sourced security testing',
+  category: 'software',
+  function: ['appsec', 'bug-bounty', 'pen-test'],
+});
+
+export const convertBounty = (
+  data: any,
+): ReturnType<typeof createIntegrationEntity> =>
```

Comment:
This is cool, hadn't seen it before. Any reason not to just `import { Entity } from '@jupiterone/integration-sdk'` instead?

Reply:
it's been the same with other integrations, so I just used the same pattern.  Does `createIntegrationEntity` always return `Entity` type?

Reply:
Yes, it always returns `Entity`.